README.md

```diff
@@ -5,12 +5,36 @@
 ### Introduction
 
 The service-catalog project is in incubation to bring integration with service
-brokers to the Kubernetes ecosystem via the
-[Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker).
-A service broker is an endpoint that manages a set of services.  The end-goal of
-the service-catalog project is to provide a way for Kubernetes users to consume
-services from brokers and easily configure their applications to use those
-services.
+brokers to the Kubernetes ecosystem via the [Open Service Broker
+API](https://github.com/openservicebrokerapi/servicebroker). A service broker
+is an endpoint that manages a set of services.  The end-goal of the service-
+catalog project is to provide a way for Kubernetes users to consume services
+from brokers and easily configure their applications to use those services,
+without needing detailed knowledge about how those services are created /
+managed.
+
+As an example:
+
+Say that a distributed application needs a message broker that hosts a set of
```

Comment:
I think the qualification of this as an example serves to communicate that it's a simple use case.  I'm fine qualifying further, but I think if you read into the use cases it becomes pretty clear.  The main intent here is to communicate the broad strokes.